Core/ContentBlockerRulesUserScript.swift

```diff
@@ -54,10 +55,10 @@ public class ContentBlockerRulesUserScript: NSObject, UserScript {
         
         guard let dict = message.body as? [String: Any] else { return }
         guard let blocked = dict[ContentBlockerKey.blocked] as? Bool else { return }
```

Comment:
@SlayterDev I'd love to get your eyes on this to make sure the fix here is sensible.

This variable was coming back true for an S3 link, and although the content blocker rules were actually ignoring it (since S3 has an exception), the privacy dashboard thought it was blocked anyway.

The fix was to update `trackerFromUrl(...)` to check that the detected tracker doesn't contain any `.ignore` rules for the URL. With that in place, an S3 URL comes in and is detected to be from Amazon, but the new check sees that S3 has an exception and overrides `blocked` being true.